mmcmds.c

```diff
@@ -5233,7 +5233,7 @@ void verifyMarkup(vstring labelMatch,
   /* This code expects a version date at the top of the file such as:
          $( set.mm - Version of 13-Dec-2016 $)
      If we later want a different format, this code should be modified. */
-  if (dateSkip == 0  && topDateSkip == 0) {
+  if (topDateCheck) {
```

Comment:
The following code expects mostRecentDate be set in line 5258.  If dateCheck is 0 this seems not be present. Verify.

Reply:
Actually, although I pushed a bugfix to guard this statement, it looks like it would work anyway (and in particular, this case also comes up when there are no statements in the file), because empty dates are treated as earliest in `compareDates`.

Reply:
I stop looking any further in such corner cases, because the behaviour is obvious, either from the code or a comment, or it is a sloppy implementation, even if it is technically ok.

The presented solution is acked.  Review is finished.